core/src/main/java/tc/oc/pgm/listeners/ChatDispatcher.java

```diff
@@ -172,54 +188,91 @@ public void sendDirect(Match match, MatchPlayer sender, Player receiver, @Text S
     send(
         match,
         manager.getPlayer(receiver), // Allow for cross-match messages
         message,
-        "[" + ChatColor.GOLD + "DM" + ChatColor.WHITE + "] -> {0}: {1}",
+        formatPrivateMessage("commands.message.to", sender.getBukkit()),
         viewer -> viewer.getBukkit().equals(sender.getBukkit()),
         null);
   }
 
+  private String formatPrivateMessage(String key, CommandSender viewer) {
+    Component action =
+        new PersonalizedTranslatable(key).getPersonalizedText().color(ChatColor.GRAY).italic(true);
+    return ComponentRenderers.toLegacyText(
+        new PersonalizedText(action, new PersonalizedText(" {0}: {1}")), viewer);
+  }
+
   @Command(
       aliases = {"reply", "r"},
       desc = "Reply to a direct message",
       usage = "[message]")
   public void sendReply(Match match, Audience audience, MatchPlayer sender, @Text String message) {
     final MatchPlayer receiver = manager.getPlayer(lastMessagedBy.get(sender.getBukkit()));
     if (receiver == null) {
-      audience.sendWarning(
-          new PersonalizedText("Did not find a message to reply to, use /msg")); // TODO: translate
+      audience.sendWarning(new PersonalizedTranslatable("commands.message.noReply"));
       return;
     }
 
     sendDirect(match, sender, receiver.getBukkit(), message);
   }
 
-  private static MatchPlayer getApproximatePlayer(Match match, String query, CommandSender sender) {
-    return StringUtils.bestFuzzyMatch(
-        query,
-        match.getPlayers().stream()
-            .collect(
-                Collectors.toMap(
-                    player -> player.getBukkit().getName(sender), Function.identity())),
-        0.75);
+  @Command(
+      aliases = {"s", "say", "send", "sendteam"},
+      desc = "Send a message to a specfic team chat",
+      usage = "[team] [message]",
+      perms = Permissions.STAFF)
+  public void sendOtherTeamChat(
+      MatchPlayer player, Match match, String targetParty, @Text String message) {
+    Party party = getApproximateParty(match, targetParty);
+
+    if (party == null) {
+      player.sendWarning(
+          new PersonalizedTranslatable(
+              "commands.chat.send.noParty", new PersonalizedText(message).color(ChatColor.AQUA)));
+      return;
+    }
+
+    if (match.isFinished() || party instanceof Tribute) {
+      player.sendWarning(new PersonalizedTranslatable("commands.chat.send.finished"));
+      return;
+    }
+
+    if (!player.isObserving()) {
+      player.sendWarning(new PersonalizedTranslatable("commands.chat.send.notObs"));
+      return;
+    }
+
+    // Same format as team chat command
+    send(
+        match,
+        player,
+        message,
+        party.getChatPrefix().toLegacyText() + PREFIX_FORMAT,
+        viewer ->
+            party.equals(viewer.getParty())
+                || (viewer.isObserving()
+                    && viewer.getBukkit().hasPermission(Permissions.ADMINCHAT)),
+        SettingValue.CHAT_TEAM);
   }
```

Comment:
I think this is neat, but we don't need it. If staff want to say something, they can just use global, rarely do you need to address a single team.